site/content/blog/2021-09-01-whats-new-in-svelte-september-2021.md

```diff
@@ -0,0 +1,80 @@
+---
+title: What's new in Svelte: September 2021
+description: StackOverflow's most loved web framework
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+This month, Svelte was [voted StackOverflow's most loved web framework](https://insights.stackoverflow.com/survey/2021#section-most-loved-dreaded-and-wanted-web-frameworks), Tan Li Hau [talked to Svelte Radio](https://share.transistor.fm/s/84c7521b) about his [Svelte-filled YouTube channel](https://www.youtube.com/channel/UCbmC3HP3FaAFdcZkui8YoMQ), and SvelteKit got closer than ever to 1.0!
+
+## New in Svelte
+- `use:actions` can now be used on `<svelte:body>` (**3.42.0**)
+- `HTMLElement`, `SVGElement` (**3.42.2**) and `BigInt` (**3.42.3**) are now known globals
+- Less code in Svelte's output thanks to the following improvements in **3.42.2**:
+  - Collapse whitespace in class and style attributes
+  - Deselect all <option>s in a <select> where the bound value doesn't match any of them
```

Comment:
```suggestion
- Deselect all `<option>`'s in a `<select>` when the bound value doesn't match any of them
```
The unescaped <option> wraps the remainder of the document in an unclosed html element, so rest of the page doesn't render (in github / codespaces / vscode).